introducing-metrics-in-innersource.md

```diff
@@ -4,105 +4,76 @@ Introducing Metrics in InnerSource
-are not focused on tracking their personal performance, but to compare
-if the initiative is currently working as expected.
+Let developers and middle managers know that these metrics or KPIs are not focused on tracking their personal performance, but to compare if the initiative is currently working as expected.
 
-Consider third party and neutral player to produce such metrics.
+Consider a third party that is seen as a neutral player to produce such metrics.
 
-Have specific training on the topic of metrics and good practices
-to use them. An example is to have a methodology to follow metrics such
-as the Goal-Question-Metric approach or the Objectives-KeyResults one.
-On the other hand, try to update the metrics used to the short-term
-and medium-term goals.
+Have specific training on the topic of metrics and good practices to use them. An example is to have a methodology to follow metrics such as the Goal-Question-Metric approach or the Objectives-KeyResults one. On the other hand, try to reflect the short-term and medium-term goals in the metrics to be used.
 
-Bring specific discussions on the metrics to be used avoiding per
-developer granularity or at least anonymizing that info.
+Metrics when published or discussed should be done so in the aggregate without refering to specific people.
 
-Produce a characterization of metrics as this might be helpful for others
-to understand and follow.
+Produce a characterization of metrics as this might be helpful for others to understand and follow.
 
-* Nearly always InnerSource is not a goal in-and-of itself but a proposal of how to improve some larger problem that the company is having. One class of metrics is around that larger goal (e.g. quality, interrupt-driven work, duplicated code, etc.) and how it is going for the company.
-* Another class of metrics is around how much InnerSourcing is happening. A raw definition of InnerSource based on code submission to a repository not owned by the submitter's team could be measurable in a few ways.
-* A third class of metrics are KPIs that we believe will improve the raw amount of InnerSourcing happening (e.g. mean-time-to-review, automated test run on PR, etc.).
+* Nearly always InnerSource is not a goal in-and-of itself but a proposal of how to improve some larger problem that the company is having. One class of metrics is around that larger goal (e.g. quality, interrupt-driven work, duplicated code, etc.) and reflects the company's progress towards it.
+* Another class of metrics is around how much InnerSourcing is happening. A raw definition of InnerSource based on the number and frequency of code submissions to a repository not owned by the submitter's team could be measurable in a few ways.
+* A third class of metrics are KPIs that we believe will improve the raw amount of InnerSourcing happening (e.g. mean-time-to-review, number of comments and/or frequency of 'changes requested' on PRs by trusted committers, automated test run on PR, etc.).
```

Comment:
> Do you have a source for these specific metrics? 
I am currently working on a script that hits our enterprise version of github. Number of comments and the frequency of 'changes requested' per PR *could* be a good observability indicator of the smoothness of the contributing process. These stats are useful when it comes to typical troubleshooting complaints from a particular 'outside' contributing group to an innersource project. An investigation into such complaints should look at to these metrics as evidence of broken contributing.md files, undocumented standards, lack of code structure among others.

Reply:
> 2\. think about which examples are the most ideal to showcase here to not by accident take in examples that could have detrimental effects for people only reading this.

I agree. The examples I cited could have detrimental effects to readers, Given this, I am happy to remove these examples. If I come across examples that can't be interpreted toward any detrimental effect, I will be happy to propose them later in a subsequent PR.

Reply:
Sounds like a good approach, let's do it that way.
If you remove the extra examples for now and refer to this discussion in the subsequent PR everyone will have the context from the old PR in the new one too.  And I can merge this right in.

As for the first comment - sounds interesting. Maybe you really want to chat up @dicortazar in our Slack on that topic. There's even a metrics related channel.

Reply:
@lenucksi Thanks. I just joined the metrics-platform slack channel. However, no activity there since mid April. @dicortazar Is there related activity in some other space/venue?